controllers/halayerset_controller_test.go

```diff
@@ -85,7 +87,8 @@ var _ = Describe("High Availability Layer Set CR", func() {
 				existingCR, err := getHALayerCR()
 				Expect(err).To(BeNil())
 				fenceAgent := v1alpha1.FenceAgentSpec{Name: updatedFenceAgentName, Type: fenceAgentType, Params: map[string]string{}}
-				existingCR.Spec.FenceAgentsSpec = []v1alpha1.FenceAgentSpec{fenceAgent}
+				existingCR.Spec.FenceAgentsSpec[0] = fenceAgent                       //replace fence agent will trigger delete and create
+				existingCR.Spec.FenceAgentsSpec[1].Params[fenceAgentPortKey] = "9223" //modify fence agent param - will trigger update
```

Comment:
What is this value, number?

Reply:
It's a string that represent the port

Reply:
Is there a reason for choosing this particular port number? I can see it is a [known port assignment for tcp/udp](https://www.speedguide.net/port.php?port=9223) .